### ISSUE TITLE
Missing or badly sized callout icons in key tables

### ISSUE THREAD

**Issue by netapp-martyh** [NetAppDocs/ontap-systems#227]
### Page URL

https://docs.netapp.com/us-en/ontap-systems/a900/chassis_replace_move_hardware.html

### Page title

Move and replace hardware - AFF A900

### Summary

Throughout the page, the callout number icons are either missing or too small.

### Public issues must not contain sensitive information

- [X] This issue contains no sensitive information.

**Comment by dougthomp** [comment]
@netapp-martyh 

Interesting issue.  Seems how the output being displayed is determinate on the browser.  I opened the topic in Edge and Firefox.  Edge displayed correctly, but Firefox did not. I don't have Chrome installed, so couldn't check if the issued showed up in chrome.

I opened this topic in Firefox and the problem in the issue showed up. Example excerpt:
![image](https://github.com/NetAppDocs/ontap-systems/assets/76076341/9ee86227-a909-47ce-a958-b9da5e9b25c4)
  I also looked at the same topic in Edge and the issue did NOT show up.  Same excerpt in Edge:
  
![image](https://github.com/NetAppDocs/ontap-systems/assets/76076341/2d5ef312-b052-475e-88a0-1c4662c48caa)
